mecsimcalc/quiz_utils.py

```diff
@@ -0,0 +1,113 @@
+import requests
+import jwt
+import time
+from datetime import datetime
+import smtplib
+from email.mime.multipart import MIMEMultipart
+from email.mime.text import MIMEText
+import logging
+
+
+# Function to append values to a Google Sheet
+def append_to_google_sheet(service_account_info, spreadsheet_id, values, range_name='Sheet1!A1', include_timestamp=True):
+    """
+    Appends values to a Google Sheet, optionally including a timestamp.
+
+    Parameters:
+        service_account_info (dict): The service account credentials.
```

Comment:
Try using the format used for other functions in the library. It follows the NumPy docstring style guide. Your current style does not display well on MecSimCalc (not sure why)

https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_numpy.html

Reply:
Hi, I follow the docstring style guide in this link(which looks like is not exact the same with other function). Let me know if there still have problem.

Reply:
It looks good, however the reason why I repeated the function in the first line is because MecSimCalc doesn't normally display the function. however VS Code does. 

Also can you update the other functions to follow the official style guide. I'm not sure why I didn't do it that way originally 

here is how it looks on VS Code (shows function + docstring)
![image](https://github.com/MecSimCalc/MecSimCalc-utils/assets/42281247/e11f04c1-4903-43d8-b8b6-345acd55a2cf)

VS MecSimCalc (Just shows docstring)
![image](https://github.com/MecSimCalc/MecSimCalc-utils/assets/42281247/16a4dadb-f861-4311-92b8-484c06516a98)

Reply:
@Jiaaming please follow @PointlessUser 's instructions. He's the original author of this repository and will give final approval for this PR to be merged. Thanks

Reply:
Yes i can update it for other functions. So do you means keep the first line in order to be viewed normally on Mecsimcalc?
I'm not sure how you preview docstring in MecSimCalc, sorry. Can you give me a bit instruction on it? So I can check it's format, thank you.

----
Currently I use `Sphinx`. My function looks like in HTML:

<img width="861" alt="Screenshot 2024-02-09 at 17 38 54" src="https://github.com/MecSimCalc/MecSimCalc-utils/assets/71547730/e3971cd6-3a06-4060-8403-f85ca706af42">

Other functions like this: 
<img width="812" alt="Screenshot 2024-02-09 at 17 39 30" src="https://github.com/MecSimCalc/MecSimCalc-utils/assets/71547730/3c68101f-2738-4c0c-b452-10d0887a1398">

Reply:
Yes keep the first line (the function) so it shows up on mecsimcalc. 

To view the docstring on mecsimcalc, you just hover over the function. example:

import mecsimcalc as msc
msc.print_plot()

if you hover your mouse over print_plot() it'll show the docstring

On Sphinx, it should look exactly like yours, except you should add the function at the top of the docstring (like mine)

Reply:
If you're still confused, I could do the changes once we accept your pull request